src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,453 @@
+    }
+
+    /**
+     * Prints out a message when the user inputs 'bye'
+     */
+    public static void bidGoodbye() {
+        String GOODBYEMESSAGE = "Alas, our lovely time together comes to an end. Au revoir!\n";
+
+        System.out.println(LINEBAR);
+        System.out.println(GOODBYEMESSAGE);
+        System.out.println(LINEBAR);
+    }
+
+    /**
+     * Prints a message denying any and all allegations of DAHNAM being a bot when prompted with 'bot?'
+     */
+    public static void denyBotNature() {
+        String denyBotNature = "No, I am definitely not a bot. Why do you ask?\n";
+
+        System.out.println(LINEBAR);
+        System.out.println(denyBotNature);
+        System.out.println(LINEBAR);
+    }
+
+    /**
+     * Enumerates through an array of tasks and prints out all tasks input by user
+     *
+     * @param taskList An array of tasks. Max size of 100
+     */
+    public static void listAllTasks(ArrayList<Task> taskList, int index) {
+        if (index == 0) {
+            System.out.println("You do not have any items in your list currently.");
+            return;
+        }
+
+        int taskNumber = 1;
+
+        System.out.println(LINEBAR);
+        //do something
+        for (int i = 0; i < taskList.size(); i++) {
+            System.out.println(taskNumber + ". " + taskList.get(i));
+            taskNumber++;
+        }
+        System.out.println(LINEBAR);
+    }
+
+    /**
+     * Notifies user of what item is added to the task list upon receiving a valid task input.
+     *
+     * @param t user input task
+     */
+    public static void echoUserInput(Task t, int taskNum) {
+        System.out.println("Added: " + t.toString() + "\n You currently have " + taskNum + " tasks");
+    }
+
+    /**
+     * Adds the task 't' specified by user into the task list
+     *
+     * @param taskList An ArrayList of tasks. Max size of 100
+     * @param t        user input task
+     */
+    public static void addToTaskList(ArrayList<Task> taskList, Task t) {
+        taskList.add(t);
+    }
+
+    /**
+     * Modifies a task and sets its boolean isDone to true. Prints out an acknowledgement after.
+     *
+     * @param userInput String that the user input
+     * @param taskList  An ArrayList of tasks.
+     * @throws NumberFormatException Thrown when the input is not a valid positive integer e.g. an alphabet
+     * @throws NullPointerException Thrown when the value inserted exceeds the number of tasks
+     * @throws IndexOutOfBoundsException Thrown when input is a negative value, or does not include a value
+     */
+    public static void completeTask(String userInput, ArrayList<Task> taskList)
+            throws NumberFormatException, NullPointerException, IndexOutOfBoundsException {
+        System.out.println(LINEBAR);
+
+        try {
+            int taskNumber = Integer.parseInt(userInput.split(" ")[TASK_NUMBER]);
+
+            //Navigate to the given index and change the sign
```

Comment:
The code logic here isn't that complex. Hence, this comment doesn't seem necessary. Alternatively, you can include it in the method's java docs.